frontend/src/v5/ui/routes/dashboard/projects/federations/createFederationForm/createFederationForm.component.tsx

```diff
@@ -55,15 +55,11 @@ export const CreateFederationForm = ({ open, onClickClose }: ICreateFederation):
 	const [modalPhase, setModalPhase] = useState('settings');
 	const [includedContainers, setIncludedContainers] = useState([]);
 
-	const closeAndReset = (): void => {
-		onClickClose();
-		reset();
-		setModalPhase('settings');
-	};
-
 	useEffect(() => {
-		if (!open) closeAndReset();
-	}, [open, reset]);
+		if (open) {
+			setModalPhase('settings');
+		} else reset();
```

Comment:
Not a change request, but you can re-write this as 
```ts 
open ? setModalPhase('settings') : reset();
```

Reply:
Done